### DISCUSSION TITLE
exp.plot_model() does not work

### DISCUSSION THREAD

**Discussion by chrismen** [pycaret/pycaret#3427] [Q&A]
The following does not work for time series models
exp.plot_model(model, plot='diagnostics', fig_kwargs={"height": 800, "width": 1000})

https://github.com/pycaret/pycaret/blob/master/tutorials/time_series/forecasting/univariate_without_exogeneous_part2.ipynb

**Comment by chrismen** [comment]
But this one works

# Plot the out-of-sample forecasts
exp.plot_model(estimator=model)

**Reply by ngupta23** [reply]
When you say, it does not work, can you elaborate? It works in the notebook that you have linked. 

Make sure you are using the latest version of the library. It should be pycaret 3.0.0 as of this time.

**Reply by chrismen** [reply]
I use the last version 3.0.0

**Comment by chrismen** [comment]
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-dcc4a928a37f> in <module>
      2 import pandas as pd
      3 from pycaret.datasets import get_data
----> 4 from pycaret.time_series import TSForecastingExperimen

ImportError: cannot import name 'TSForecastingExperimen' from 'pycaret.time_series' (/home/ubuntu/.local/lib/python3.7/site-packages/pycaret/time_series/__init__.py)

**Reply by ngupta23** [reply]
That has nothing to do with the `plot_model` line that you pointed out. 

The import spelling is incorrect in your example. You are missing a "t" at the end.

```
from pycaret.time_series import TSForecastingExperiment
```

not

```
from pycaret.time_series import TSForecastingExperime
```
